finance/sc-im/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        andmarti1424 sc-im 0.8.2 v
+maintainers         @nkorth openmaintainer
+license             BSD
+categories          finance
+description         Spreadsheet Calculator Improvised
+long_description    An ncurses spreadsheet program for terminal
+
+platforms           darwin
+
+github.tarball_from archive
+checksums           rmd160  afabf063feda1800fd07e6ee13b4036c7d5a200f \
+                    sha256  7f00c98601e7f7709431fb4cbb83707c87016a3b015d48e5a7c2f018eff4b7f7 \
+                    size    1619617
+
+worksrcdir          ${distname}/src
+destroot.destdir-append prefix=${prefix}
+
+# Define GNUPLOT even if it's not present. It's not actually a build dependency,
+# it's just used via system() when the user runs one of sc-im's plot commands.
+# This define lets the user always use those commands, and they'll get an error
+# message if they don't have gnuplot installed.
+build.env-append    CFLAGS=-DGNUPLOT
```

Comment:
looking at the build log there are references to `/usr/local` which isn't correct:
`-DHELP_PATH=\"/usr/local/share/sc-im\" -DLIBDIR=\"/usr/local/share/doc/sc-im\"`

Indeed, if you type `help` at the prompt after starting the program you'll get the message "help file not found".

So you probably should specify these variable as `build.env-append` or some other way so that they refer to `${prefix}/share/sc-im` that is where the files are being installed by MacPorts.

Reply:
Thanks for catching that. I think I fixed all the remaining references to /usr/local.